### ISSUE TITLE
Finish phase 1 dashboard work

### ISSUE THREAD

**Issue by briri** [CDLUC3/dmptool#574]
Add the final elements of the DMP Upload dashboard:
- [x] Slide out information card
- [ ] Filters

@fraserclark please make the PR to the `v5` branch when you're ready

**Comment by briri** [comment]
Sorry for the delay in getting a sample of what the related works JSON looks like to you.

Here is a recent DMP one of our pilot project partners uploaded through the phase 1 functionality. It has 8 related works that need review: https://api.dmphub.uc3prd.cdlib.net/dmps/10.48321/D13BEA529C

The related works can be found in the `dmphub_modifications` array

**Comment by fraserclark** [comment]
@briri I've added the pull request now 

https://github.com/CDLUC3/dmptool/pull/589

**Comment by briri** [comment]
@fraserclark I merged your PR and then needed to make a few minor adjustments to the way related_works were being checked/handled and also with the grant_id. I think maybe you had old data on your local or were working with some older registered plans that had some outdated metadata.

For reference, those changes are in my PR #594 no need for you to do anything with it, just an FYI

**Comment by mariapraetzellis** [comment]
My notes from testing the slider: 

1. I have 3 small formatting & text changes:
- Increase the margins of the Grant ID field. It is currently displaying too close to the right margin of the slider.
<img width="336" alt="Screen Shot 2024-05-21 at 1 15 26 PM" src="https://github.com/CDLUC3/dmptool/assets/27511575/06f8f964-4f5f-42a0-9ddc-d2899e2e8530">

- Change the field title of the "Last Updated" field to "Last Edited" to match the wording from the table
- Remove extra “-“ in the Project Dates field: “Start: 2021-01-15 **-** End: 2024-12-31”

2.  Is it possible to add an "X" to close the slider window so you don’t have to scroll to the bottom in order to close the window?

3. I found one possible bug. I created a new plan via the upload feature but did not generate a DMP ID. Howver, there is still a DMP ID included in the slider. https://dmptool-stg.cdlib.org/dashboard/dmp/20240521-2394df1c2e33
